support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/MilestoneStep.java

```diff
@@ -0,0 +1,44 @@
+package org.jenkinsci.plugins.workflow.support.steps;
```

Comment:
FYI in #369 I expect to move `StageStep` out of its current location, so expect this to have a new home too.